packages/help-center/src/components/help-center-contact-form.tsx

```diff
@@ -316,6 +318,15 @@ export const HelpCenterContactForm = () => {
 						initialChatMessage += `<strong>Automated AI response from ${ gptResponse.source } that was presented to user before they started chat</strong>:<br />`;
 						initialChatMessage += gptResponse.response;
 					}
+					if ( wapuuFlow ) {
+						initialChatMessage += '<br /><br />';
+						if ( supportChatId ) {
+							initialChatMessage += `<strong>Wapuu chat reference: ${ supportChatId }</strong>:<br />`;
+						} else {
+							initialChatMessage += `<strong>Wapuu chat referene is not available </strong>:<br />`;
+						}
+						initialChatMessage += 'User was chatting with Wapuu before they started chat<br />';
+					}
```

Comment:
```suggestion
					if ( wapuuFlow && supportChatId ) {
						initialChatMessage += '<br /><br />';
						initialChatMessage += `<strong>User chatted with Wapuu before escalating: see ${ supportChatId }</strong>:<br />`;
					}
```
Correct me if I'm wrong, but if `supportChatId` is empty, then they probably did _not_ chat with Wapuu, right? I'm not sure if there's a flow like that in Calypso right now, but of course it's better to be on the safe side.

Either way, I'll try to move these messages to the backend, when we'll be making the link between the AI chat and Zendesk 🙇 But for now, some rudimentary message is good to have.

Reply:
I preferred to be safe than sorry. It was hard for me to think about a case where the chat_id is not defined. But then I found it.

If we ask Wapuu anything, and AI services are down, the chat_id might be not created. That way, users can go directly to the Wapuu chat, without having a real chat (in case of an error, we offer the customer the old flow)